Automattic/jetpack
# Docs: Update command to run server-rendered fixture tests.

#### Changes proposed in this Pull Request:
* Just updating the command to run server-rendered fixture tests since the old one doesn't work anymore.

#### Jetpack product discussion
Not needed.

#### Does this pull request change what data or activity we track or use?
None

#### Testing instructions:
* Run `pnpm docker:phpunit -- --testsuite=extensions` locally, it will say that it couldn't find the script.
* Run `jetpack docker phpunit -- --testsuite=extensions` locally. It should work.
